kolibri/core/auth/models.py

```diff
@@ -639,7 +639,7 @@ def create_user(self, username, email=None, password=None, **extra_fields):
         if extra_fields.get("facility") is None:
             extra_fields["facility"] = Facility.get_default_facility()
         if self.filter(
-            username__iexact=username, facility=extra_fields["facility"]
+            username=username, facility=extra_fields["facility"]
```

Comment:
This should not be updated - the intention was for the bulk import flow to match the enforcement that happens here, not for this enforcement to be removed.